frappe/app.py

```diff
@@ -132,18 +130,17 @@ def application(request: Request):
 
```

Comment:
We don't rollback anymore on previous line. 

We used to before because of how `finally:` behaves.

Reply:
oh yes, I missed it.

Reply:
[`a524fcf` (#32980)](https://github.com/frappe/frappe/pull/32980/commits/a524fcfee644f1dc419ee0622b4551449c7d4fb8)